decayAmplitude/waveDescription.cc

```diff
@@ -536,6 +636,54 @@ waveDescription::constructAmplitude(isobarAmplitudePtr&           amplitude,
 }
 
 
+namespace {
+
+
+	string
+	nameForMassDependence(const particlePtr          P,
+	                      const isobarDecayVertexPtr vertex)
+	{
+		ostringstream name;
+		if (vertex->massDependence() && vertex->massDependence()->name() == "binned") {
```

Comment:
get rid of the ifs by using massDependence()->name() and by adding a virtual member function to massDependence and the child classes that returns the necessary string

extend this so that is can also be used to distinguish waves with different isobar parametrizations